src/Components/book.css

```diff
@@ -0,0 +1,227 @@
+  border: none;
+  color: #4386bf;
+  font-size: 0.875rem;
+  font-weight: 300;
+  align-items: center;
+  cursor: pointer;
+}
+
+.circle-container {
+  height: 80px;
+  width: 80px;
+  background-color: #e8e8e8;
+}
+
+.first-circle {
+  height: 80px;
+  width: 80px;
+  border: 8px solid white;
+  border-radius: 50%;
+}
+
+.second-circle {
+  height: 80px;
+  width: 80px;
+  border: 8px solid black;
+  border-radius: 40%;
+}
+
+.Progress-Detail {
+  display: flex;
+  flex-direction: column;
+}
+
+.Progress-Container {
+  display: flex;
+}
+
+.circle-wrap {
+  width: 4.25rem;
+  height: 4.25rem;
+  background: #ddd;
+  border-radius: 50%;
+  border: 1px solid #cdcbd0;
+}
+
+.mask .fill {
+  clip: rect(0, 2.125rem, 4.25rem, 0);
+  background-color: #227ded;
+}
+
+.mask.full,
+.circle .fill {
+  animation: fill ease-in-out 3s;
+  transform: rotate(135deg);
+}
+
+.circle-wrap .circle .mask,
+.circle-wrap .circle .fill {
+  width: 4.25rem;
+  height: 4.25rem;
+  position: absolute;
+  border-radius: 50%;
+}
+
+.circle-wrap .circle .mask {
+  clip: rect(0, 4.25rem, 4.25rem, 2.125rem);
+}
+
+.circle-wrap .inside-circle {
+  width: 3.458rem;
+  height: 3.458rem;
+  border-radius: 50%;
+  background: white;
+  margin-top: 0.3968rem;
+  margin-left: 0.3968rem;
+  color: #1e51dc;
+  position: absolute;
+  z-index: 100;
+}
+
+@keyframes fill {
+  0% {
+    transform: rotate(0deg);
+  }
+
+  100% {
+    transform: rotate(135deg);
+  }
+}
+
+.-Percent-Complete {
+  width: 2.938rem;
+  height: 2.438rem;
+  margin: 0.063rem 7.063rem 0 0.688rem;
+  font-family: Montserrat, sans-serif;
+  font-size: 2rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.Completed {
+  width: 5rem;
+  height: 1.125rem;
+  margin: 0.438rem 4.938rem 0.75rem 0.75rem;
+  opacity: 0.5;
+  font-family: Montserrat, sans-serif;
+  font-size: 0.875rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.vertical-line-two {
+  width: 0.25rem;
+  height: 6rem;
+  background-color: #e8e8e8;
+  margin: 1rem;
+}
+
+.button-div {
+  display: flex;
+  flex-direction: column;
+  width: 14rem;
+}
+
+.button-div-heading {
+  font-family: "Roboto Slab", serif;
+  font-weight: 300;
+  color: #121212;
+  font-size: 0.813rem;
+  opacity: 0.5;
+}
+
+.button-div-middle-txt {
+  font-family: "Roboto Slab", serif;
+  font-weight: 300;
+  color: #121212;
+  margin-top: -10px;
+  font-size: 18px;
+}
+
+.button-div-btn {
+  cursor: pointer;
+  border: none;
+  background-color: #0290ff;
+  border-radius: 3px;
+  padding: 8px 24px;
+  font-family: "Roboto Slab", serif;
+  font-weight: 300;
+  letter-spacing: 0.5px;
+  color: #fff;
+}
+
+.main-wrapper {
+  display: flex;
+  background-color: #fff;
+  border-radius: 4px;
+  border: 1px solid #e8e8e8;
+  padding: 2rem;
+  justify-content: space-between;
+}
```

Comment:
- [ ] I would suggest you decrease the height of the container to match the design
- [ ] I would suggest you align the completed section in the middle and kindly decrease the line as shown in the figure
- [ ] I would suggest use the same color and font to stick on the design

### Your design
![your](https://user-images.githubusercontent.com/81727988/181935162-a7bf0f6f-b845-4f2c-8cd9-e852c7ce755a.png)
### Figma design
![fffffffff](https://user-images.githubusercontent.com/81727988/181935166-b28661cf-fa4e-42dd-81e4-4196f77bbb2d.png)